runtime/interpreter/interpreter.go

```diff
@@ -4048,6 +4077,9 @@ func (interpreter *Interpreter) ValidateAtreeValue(v atree.Value) {
 			return info.Equal(other.(StaticType))
 		case compositeTypeInfo:
 			return info.Equal(other)
+		case EmptyTypeInfo:
```

Comment:
Storage maps are atree ordered maps without any type info. They map storage paths to arbitrarily typed values, so static type information is useless